apps/web/next.config.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
-const withTM = require("@vercel/edge-functions-ui/transpile")(["react-timezone-select"]);
+const withTM = require("@vercel/edge-functions-ui/transpile")([]);
```

Comment:
We no longer need to transpile react-timezone-select? In that case we can probably remove the entire HOC